examples/new_workbench_project_from_volume_mesh.py

```diff
@@ -0,0 +1,15 @@
+import flow360 as fl
+from flow360.component.volume_mesh import VolumeMeshV2
+
+fl.Env.dev.active()
+
+# volume mesh
+volume_mesh_draft = VolumeMeshV2.from_file(
+    "data/om6wing/mesh.lb8.ugrid",
```

Comment:
same here

Reply:
Fixed in [6cece60](https://github.com/flexcompute/Flow360/pull/470/commits/6cece60781cc6aba8fc1ca2fdedd6b33711331be)